app/v2/notifications/post_notifications.py

```diff
@@ -348,10 +406,10 @@ def process_precompiled_letter_notifications(*, letter_data, api_key, template,
     return notification
 
 
-def get_reply_to_text(notification_type, form, template):
+def get_reply_to_text(notification_type, form, template, form_field=None):
     reply_to = None
     if notification_type == EMAIL_TYPE:
-        service_email_reply_to_id = form.get("email_reply_to_id", None)
+        service_email_reply_to_id = form.get(form_field or "email_reply_to_id")
```

Comment:
I guess this is the fix for the "reply to address not being set properly" bug?

Reply:
Actually the fix is to set the `sender_id` in the Job model. `job/rest.py` already does it, updated tests to prove that now this is saved in the database and the process task uses the `sender_id` column from the job if it's set https://github.com/cds-snc/notification-api/pull/1305/files#diff-e9561f078706a45099a66d687107104b4212f64960c1d74166dfab7b87f91ae4R181
https://github.com/cds-snc/notification-api/pull/1305/files#diff-3d49ca02b37d5d2094c793bf77b4e0d7fdddf6938c4bffe931bd3efb6df2d744R123